### DISCUSSION TITLE
Feat: dark mode

### DISCUSSION THREAD

**Discussion by brendanfalk** [aws/codewhisperer-command-line-discussions#4] [Feature request]
### Checks

- [X] I could not find an existing setting that solved my issue
- [X] I searched [the discussions in this repo](https://github.com/aws/codewhisperer-command-line-discussions/discussions?q=) and there are no duplicates of my feature request

### Feature Details

The users want dark mode in the dashboard!! So that's what we will give them

**Comment by superbryntendo** [comment]
Just pushed a PR for this internally. It's coming!

**Reply by superbryntendo** [reply]
Merged and will arrive in the next release!

**Comment by grant0417** [comment]
This is released in 0.13.0
